src/jdk.incubator.code/share/classes/jdk/incubator/code/internal/ReflectMethods.java

```diff
@@ -2729,11 +2692,48 @@ boolean isThisOrSuper(JCExpression expression) {
         }
     }
 
+    private static class OpMap extends LinkedHashMap<String, Op> {
```

Comment:
Make it a record holding a `SequencedMap`, with a constructor that creates a linked hash map instance. You can also include methods that defer to the map if needed. Then you don't need the serialVersionUID, and if the compiler allows it you can pattern match on the record's map component to directly operate on the map in `genCode`.